scripts/example/jupyterlab_version/l8_cloud_train_network_mod1.yaml

```diff
@@ -0,0 +1,120 @@
+train:
+  loss: sparse_categorical_crossentropy
+  metrics:
+    - sparse_categorical_accuracy
+  network:
+    layers:
+     - Input:
+         shape: [~, ~, num_bands]
+     - Conv2D:
+          filters: 16
+          kernel_size: [3, 3]
+          padding: same
+     - BatchNormalization:
```

Comment:
Instead of mod1 having everything, might be less confusing to have the old file, then just have changes in mod1, and use two --config options to overwrite those specific options? Not sure about this. Or have a more descriptive name than mod1 :)